frontend/src/components/grantaccess/form/ForestClientInput.vue

```diff
@@ -30,7 +30,9 @@ const forestClientNumberVerifyErrors = ref([] as Array<string>);
 
 const verifyForestClientNumber = async (forestClientNumbers: string) => {
     forestClientNumberVerifyErrors.value = [];
-    let forestNumbers = forestClientNumbers.split(/[ ,]+/);
+    // regex to match commas with optional space
+    let regex = /\s*,\s*|\s+/
```

Comment:
Hi Jota,  I think this line gives us a security warning in sonar cloud, do you mind take a look? thanks!

Reply:
Hello, Catherine! I'll try to fix it but might need some help